### DISCUSSION TITLE
How to turn off emails for "Pre-release continuous"?

### DISCUSSION THREAD

**Discussion by fatrabbit-la** [artisan-roaster-scope/artisan#790] [General]
I get numerous emails titled, "Pre-release continuous - Continuous build". I definitely want to receive updates about new version releases but I don't think these particular emails are relevant. How do I turn them off?

**Comment by roasterdave** [comment]
You need to turn off some of the notifications from Github.  They are found by clicking the pull-down next to **_Unwatch_** near the upper right of the page.  More details that should help can be [found here](https://artisan-scope.org/discussions/).

**Comment by fatrabbit-la** [comment]
Thanks Dave. Can you clarify what all these are? Aside from ‘Discussions’, I don’t know what the other options are and therefore which I want to get notifications for.

Like for ‘Releases’ - are those the pre-release continuous I want to turn off? I don’t want to miss notification of an actual release to update my software. ‘Issues’ and ’Security alerts’ sounds important but maybe not relevant to a general user such as myself? No idea what ‘Pull requests’ is.

Appreciate it.

Dan

Fat Rabbit Coffee
***@***.*** ***@***.***>
www.fatrabbitcoffee.com <http://www.fatrabbitcoffee.com/>

**Reply by poundy** [reply]
These are all standard Github classifications, nothing special about Artisan.  If you want to be notified of releases, then you (unfortunately) get notified about pre-release continuous builds as well. There's such a high noise-to-signal ratio on that that I'd suggest, if you don't want to set up a rule to delete them automatically or clean them up manually, that you don't subscribe to release notifications, you'll get notified through other channels when proper releases are made. Pull requests - unless you're writing code for Artisan you probably won't need that. Issues are sometimes lodged by users instead of discussing things first (in discussions) but are actual bug reports that can get addressed through code changes. There's not that many, so not much noise if you want to see what happens.  Security alerts also end up in bugs and eventually releases, so again if you're mainly interested in the community discussion and not in the code side of things, you can avoid that too.

**Comment by fatrabbit-la** [comment]
Perfect - thanks for clarifying!

Fat Rabbit Coffee
***@***.*** ***@***.***>
www.fatrabbitcoffee.com <http://www.fatrabbitcoffee.com/>
